doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -87,17 +83,72 @@ The following block of code shows how you can use the API to establish an LTE co
    }
 
 The code block demonstrates how you can use the library to asynchronously set up an LTE connection.
-For more information on the callback events received in :c:type:`lte_lc_evt_handler_t` and data associated with each event, see the documentation on :c:struct:`lte_lc_evt`.
 
-The following list mentions some of the information that can be extracted from the received callback events:
+Additionally, to enable specific functionalities and receive specific events from the library, you must enable the corresponding modules through their respective Kconfig options:
+
+Connection Parameters Evaluation:
+  Use the :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option to enable the following functionality related to Connection Parameters Evaluation:
+
+  * :c:func:`lte_lc_conn_eval_params_get`
+
+eDRX (Extended Discontinuous Reception):
+  Use the :kconfig:option:`CONFIG_LTE_LC_EDRX` Kconfig option to enable all the following functionalities related to eDRX:
+
+  * :c:enumerator:`LTE_LC_EVT_EDRX_UPDATE` events
+  * :c:func:`lte_lc_ptw_set`
+  * :c:func:`lte_lc_edrx_param_set`
+  * :c:func:`lte_lc_edrx_req`
+  * :c:func:`lte_lc_edrx_get`
+  * :kconfig:option:`LTE_EDRX_REQ`
```

Comment:
Needed to get hyperlink to work:
```suggestion
  * :kconfig:option:`CONFIG_LTE_EDRX_REQ`
```

Fix all kconfigs in this chapter.

Reply:
Very good catch. I totally overlooked these.

Reply:
Thanks!